module-4/backend/week_four.md

```diff
@@ -4,8 +4,48 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-TBD  
+1. What's your favorite project management tool?
+
+I like both waffle and pivotal, pros and cons with both. Need more reps to have a matured opinion
+
+2. Why do we create staging environments?
+
+Mimic production
+
+3. What are the characteristics of a good README (in your opinion)?
+
+Clear and concise setup procedure, docs, not dry content, add a little flavor
+
+4. What's one main improvement you're going to make to your code regarding accessibility issues?
+
+Writing good alt-text content
+
+5. What are some basic security concerns to be aware of when building applications?
+
+Injection, not exposing id's for database items, requiring tokens for api's.
+
+6. Why is continuous integration helpful/important?
+
+Lots of small updates are more easily managed/tested
```

Comment:
Also keep in mind that this can standardize styling and testing protocols for deployment.